versions.json

```diff
@@ -10,5 +10,5 @@
   "kontrol": "1.0.53",
   "just": "1.34.0",
   "binary_signer": "1.0.4",
-  "semgrep": "1.90.0"
+  "semgrep": "1.96.0"
```

Comment:
we don't need this change.

Reply:
removed it (git reset and force pushed the new commit without this change: [008c81b](https://github.com/ethereum-optimism/optimism/pull/11606/commits/008c81bdbea0c5bd95b23f6bc3ad769548a27bcb))